test/functional/examples/change_streams.test.js

```diff
@@ -4,7 +4,9 @@
 const setupDatabase = require('../shared').setupDatabase;
 const expect = require('chai').expect;
 
-describe('examples(change-stream):', function () {
+// TODO: NODE-3819: Unskip flaky MacOS/Windows tests.
+const maybeDescribe = process.platform !== 'linux' ? describe.skip : describe;
```

Comment:
Just wanted to double check since the comment says MacOS/Windows that this specific one is only MacOS related

Reply:
I actually had it failing on Windows as well so switched the logic to "not linux"